interface/forms/dictation/new.php

```diff
@@ -1,29 +1,51 @@
-<!-- Form generated from formsWiz -->
 <?php
+/** 
+ *  Dictation Form
+ * 
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @copyright Copyright (c) 2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * 
+ */
 
-
-
+use OpenEMR\Core\Header;
 
 include_once("../../globals.php");
 include_once("$srcdir/api.inc");
 formHeader("Form: dictation");
 $returnurl = 'encounter_top.php';
 ?>
-<html><head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<html>
+<head>
+  <title>Dictation</title>
+   <?php Header::setupHeader(); ?>
+   <script type="text/javascript" src="<?php echo $webroot."/public/assets/ckeditor-4-7-0/ckeditor.js" ?>"</script>
+   <script src="<?php echo $webroot."/public/assets/ckeditor-4-7-0/js/sample.js" ?>" type="text/javascript"></script>
+   <link href="<?php echo $webroot."/public/assets/ckeditor-4-7-0/css/samples.css"; ?>" rel="stylesheet" type="text/css" />
 </head>
 <body class="body_top">
-<form method=post action="<?php echo $rootdir;?>/forms/dictation/save.php?mode=new" name="my_form">
-<span class="title"><?php echo xlt('Speech Dictation'); ?></span><br><br>
-<span class=text><?php echo xlt('Dictation: '); ?></span><br><textarea cols=80 rows=24 wrap=virtual name="dictation" ></textarea><br>
-<span class=text><?php echo xlt('Additional Notes:'); ?> </span><br><textarea cols=80 rows=8 wrap=virtual name="additional_notes" ></textarea><br>
-<br>
-<a href="javascript:top.restoreSession();document.my_form.submit();" class="link_submit">[<?php echo xlt('Save'); ?>]</a>
-<br>
-<a href="<?php echo "$rootdir/patient_file/encounter/$returnurl";?>" class="link"
- onclick="top.restoreSession()">[<?php echo xlt('Don\'t Save'); ?>]</a>
-</form>
+<div class="container">
+   <div class="page-header">
+      <h1><?php echo xlt('Speech Dictation'); ?></h1><br><br>
+   </div>
+
+  <form method=post action="<?php echo $rootdir;?>/forms/dictation/save.php?mode=new" name="my_form">
```

Comment:
forms needs to deal with session leak issue (especially since it was removed from the Save link):
``````
<form method=post action="<?php echo $rootdir;?>/forms/dictation/save.php?mode=new" name="my_form" onsubmit="return top.restoreSession()">
``````
More information on this can be found here:
http://www.open-emr.org/wiki/index.php/OpenEMR_System_Architecture#PHP_Sessions_and_Browser_Windows